dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -151,16 +201,23 @@ private static void AddOccurrence(List<TimelineEntryViewModel> timelineEntries,
             });
         }
 
-        private static async Task<string> ExecuteRestRequest(Guid personId, Guid recordId, Uri uri, HttpMethod requestMethod, string apiVersion)
+        private static async Task<string> ExecuteRestRequest(Guid personId, Guid recordId, Uri uri, HttpMethod requestMethod, string apiVersion, string requestBody = null)
         {
             // Create HTTP transport objects
```

Comment:
Not a change from this review but this comment reads funny (`// Create HTTP transport objects`). Particularly the word _transport_.